comp/core/config/params_darwin.go

```diff
@@ -0,0 +1,11 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+const (
```

Comment:
Why adding this file instead of using the value in `"github.com/DataDog/datadog-agent/cmd/agent/common"`?
This comment applied to other `params_XXX.go` files.

Reply:
Importing `github.com/DataDog/datadog-agent/cmd/agent/common` comes with a lot of unwanted dependencies. If you are interested, I can post the full list.

Reply:
An alternative solution would be to create  a new package `github.com/DataDog/datadog-agent/cmd/agent/common/defaultpaths` that contains only `DefaultConfPath`, `DefaultLogFile`, `DefaultDCALogFile`, `DefaultJmxLogFile`,  `DefaultCheckFlareDirectory` and `DefaultJMXFlareDirectory`. It would have the advantage of having  `DefaultConfPath` defined at a single location and `github.com/DataDog/datadog-agent/cmd/agent/common/defaultpaths` wouldn't have many dependencies.